storage/migrations/mysql/5_scheduled_campaigns.sql

```diff
@@ -0,0 +1,15 @@
+-- +migrate Up
+
+CREATE TABLE IF NOT EXISTS `scheduled_campaigns`
+(
+    `id`            varbinary(27)       primary key,
+    `campaign_id`   integer unsigned    NOT NULL,
+    `scheduled_at`  datetime(6)         NOT NULL,
+    `created_at`    datetime(6),
```

Comment:
not null as well for the timestamps.

Reply:
@Dzalevski we need to change this on `shared/scheduled-campaigns` branch